popup/search-results.css

```diff
@@ -0,0 +1,129 @@
+#disable-all-wrapper {
+  border-bottom: solid 1px #000;
+  margin-bottom: 5px;
+}
+
+#searchResults,
+#find-styles {
+  display: block;
+}
+
+#searchResults.hidden,
+#searchResults-error.hidden,
+#find-styles.hidden,
+#open-search.hidden,
+.searchResult-customize.hidden {
+  display: none;
+}
+
+#searchResults-error {
+  background-color: rgba(255, 0, 0, 0.4);
+  font-weight: bold;
+  padding: 5px;
+  text-align: center;
+  margin-right: 20px;
+}
+
+#searchResults, #searchResults-list {
+  max-width: 200px;
```

Comment:
I think the width shouldn't be constrained as the white area on the right looks weird with the title and description cut off, especially if the popup width was customized to a bigger value. Instead let the description/installation block jump to the right when there's space e.g. if the popup width is bigger than 360px.